doc/source/user/quickstart.rst

```diff
@@ -884,6 +884,12 @@ The ``copy`` method makes a complete copy of the array and its data.
            [   8,   10,   10,   11]])
 
 
+Sometimes ``copy`` should be called after slicing if the original array is not required anymore::
+
+    >>> a = np.arange(int(1e8))  # suppose `a` is a huge intermediate result and
+    >>> result = a[:100].copy()  # we only want a small part of `a`. We can make a deep copy so that
+    >>> del a  # the memory of `a` can be released. Otherwise `a` is referenced by `result` and persists in memory.
+
```

Comment:
Not a big fan of mixing multiline comments and code. And `a` should have double backticks since it is not a reference.